client/my-sites/plans-features-main/index.jsx

```diff
@@ -277,7 +280,7 @@ export class PlansFeaturesMain extends Component {
 	renderToggle() {
 		const { displayJetpackPlans, withWPPlanTabs, countryCode } = this.props;
 		if ( displayJetpackPlans ) {
-			if ( abtest( 'onlyJetpackMonthly', countryCode ) === 'monthlyOnly' ) {
+			if ( countryCode && abtest( 'jetpackMonthlyPlansOnly', countryCode ) === 'monthlyOnly' ) {
```

Comment:
This is the condition that could trigger the toggle flash that @tyxla reports. If `countryCode` is not yet loaded (i.e., is `null`), then `this.getIntervalTypeToggle()` is always called.

At some places, `countryCode` is checked in one of parent components and a placeholder is rendered instead. But not everywhere.

Reply:
right, talking with @tyxla we realized that I was missing handling the placeholder conditional for /jetpack/plans/store . I've added it in the last commit :+1: